dashboards.go

```diff
@@ -125,11 +125,24 @@ func (client *Client) UpsertDashboard(ctx context.Context, folder *Folder, build
 		return nil, err
 	}
 
+	alertPanelNames := make(map[string]struct{})
```

Comment:
We know the capacity and can init the map with it: `make(map[string]struct{}, len(alerts))`